test/functionalTest/cases/1156_qfilters_and_compounds/qfilters_and_compounds_equals_null.test

```diff
@@ -0,0 +1,221 @@
+ Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Sure, not a bad idea.
I fix it as how it must be after implementing NULL (Issue 2359) and I leave it as DISABLED